tests/model_fields/test_jsonfield.py

```diff
@@ -298,6 +298,14 @@ def test_realistic_object(self):
         obj.refresh_from_db()
         self.assertEqual(obj.value, value)
 
+    def test_field_api(self):
+        field = JSONModel._meta.get_field("value")
+        with mock.patch.object(
+            field, "get_prep_value", wraps=field.get_prep_value
+        ) as spy_get_prep_value:
+            JSONModel.objects.create(value={})
+            spy_get_prep_value.assert_called_once_with({})
+
```

Comment:
I would suggest we don't use mocks to test the added code, but instead we use some more meaningful assertions that would ensure that the possible interactions between the existing code in `get_db_prep_value` and the new fragment work as expected. For example, define a child class of `JSONField`, with a simple but noticeable `get_prep_value` implementation, and then do queries with it. Perhaps you could build some tests following your use case?

Also, I believe that tests for this adding would be best placed under `TestQuerying`.

Reply:
I moved it to `TestMethods` since the new test implementation directly call `get_db_prep_value`